joy/package.xml

```diff
@@ -26,7 +26,10 @@
   <depend>roscpp</depend>
   <depend>diagnostic_updater</depend>
   <depend>sensor_msgs</depend>
+  <build_depend>hidapi</build_depend>
```

Comment:
Is the `hidapi` a dependency for SDL2?

Reply:
Good catch. This was an intermediate code I was working on before switching to SDL2. I'll update.